src/strategies/revest/index.ts

```diff
@@ -0,0 +1,153 @@
+    // @ts-ignore
+    eip1155OwnersParams.accounts.__args.block = { number: snapshot };
+  }
+  try {
+    let idsToCheck = new Set<string>();
+
+    const result = await subgraphRequest(
+      SUBGRAPH_URL[network],
+      eip1155OwnersParams
+    );
+    result.accounts.forEach(element => {
+      element.relBal = element.balances.filter((balance) => {
+        const isRightAddress =
+          balance.token.registry.id.toLowerCase() === options.address.toLowerCase() && balance.value != '0';
+        if(isRightAddress) {
+          idsToCheck.add(balance.token.identifier);
+        }
+        return isRightAddress;
+      });
+    });
+    let ids = Array.from(idsToCheck);
+    const response = await multicall(
+      network,
+      _provider,
+      abi2,
+      ids.map((id: any) => [options.staking, 'getDisplayValues', [id, 0]]),
+      {blockTag}
+    );
+
+    let fnfts = Object.fromEntries(
+      response.map((value, i) => [
+        ids[i],
+        defaultAbiCoder.decode(['uint', 'uint'], value[0])[0]
+    ]));
+    Object.keys(fnfts).forEach(element => {
+      if(fnfts[element].eq('0')) {
+        delete fnfts[element];
+      }
+    })
+    ids = Object.keys(fnfts);
+    const response2 = await multicall(
+      network,
+      _provider,
+      abi,
+      ids.map((id: any) => [options.tokenVault, 'getFNFT', [id]]),
+      {blockTag}
+    );
+    let completeFNFTs = Object.fromEntries(
+      response2.map((value, i) => [
+        ids[i],
+        {allocPoints: fnfts[ids[i]], isRVST: value[0].asset.toLowerCase() == options.token.toLowerCase()}
+    ]));
+    let rewards = await multicall(
+      network,
+      _provider,
+      abi2,
+      [""].map((id: any) => [options.staking, 'rewardsHandlerAddress', []]),
+      {blockTag}
+    );
+    rewards = rewards[0][0];
+    let allocLP = await multicall(
+      network,
+      _provider,
+      abi3,
+      [[rewards, 'totalLPAllocPoint', []], [rewards, 'totalBasicAllocPoint', []]],
+      {blockTag}
+    );
+    let allocToken = allocLP[1][0];
+    allocLP = allocLP[0][0];
+    
+    //allocToken = allocToken[0][0];
+
+    let finalResult = {};
+    result.accounts.forEach(account => {
+      account.relBal.forEach(relBalEle => {
+        if(completeFNFTs.hasOwnProperty(relBalEle.token.identifier)) {
+          let score = completeFNFTs[relBalEle.token.identifier].allocPoints.mul(PRECISION).div(completeFNFTs[relBalEle.token.identifier].isRVST ? allocToken : allocLP);
+          if(finalResult.hasOwnProperty(getAddress(account.id))) {
+            finalResult[getAddress(account.id)].add(score);
+          } else {
+            finalResult[getAddress(account.id)] = score;
+          }
+        }
+      });
+    });
+    let returnVals = {};
+    Object.keys(finalResult).forEach(element => {
+      returnVals[element] = parseInt(finalResult[element].toString(), 10); 
+    });
+    console.log(returnVals);
+    return returnVals;
+  } catch (err) {
+    return {};
+  }
```

Comment:
Hi, It should still throw an error, `{}` may mean there is 0 voting power for all addresses, maybe remove the try-catch, should be good to go :)